earth_enterprise/src/version.txt

```diff
@@ -1 +1 @@
-5.2.2
\ No newline at end of file
+5.2.2
```

Comment:
Not related to this PR, but I wonder if this was checked in by accident.

Reply:
Yeah, I originally deleted it by accident iirc